SConstruct

```diff
@@ -120,6 +120,18 @@ elif env["platform"] == "android":
     env.Append(LIBPATH=[env['fmod_lib_dir'] + 'android/core/lib/' + arch_dir, env['fmod_lib_dir'] + 'android/studio/lib/' + arch_dir])
     env.Append(LIBS=[libfmod, libfmodstudio])
 
+elif env["platform"] == "web":
+    web_path = 'web/%s/lib/fastcomp/bitcode/'
+    #libfmod_bindings = os.path.join(fmod_lib_dir, web_path % 'core', 'fmod%s_bindings.bc' % lfix)
```

Comment:
This currently causes:
```
wasm-ld: error: duplicate symbol: JSFMOD::gSoundMap
>>> defined in C:\Users\mgonz\AppData\Local\Temp\emscripten_temp_vcebpnkd\fmodL_bindings_0.o
>>> defined in C:\Users\mgonz\AppData\Local\Temp\emscripten_temp_vcebpnkd\fmodstudioL_bindings_2.o
```

Reply:
Shoutout to @alessandrofama for finding this in the FMOD docs:

> The FMOD Studio API files already contain the Core API, so you do not need to include the Core API files as well.

So we only have to include the fmodstudio dependency.